docs/source/installing.rst

```diff
@@ -5,17 +5,22 @@ Quick Start
 -----------
 
 Use the following instructions to get up and running with ``unstructured`` and test your
-installation.
+installation. NOTE: We do not currently support python 3.11; please use an older version.
```

Comment:
@MthwRobinson is this no longer the case, per https://github.com/Unstructured-IO/unstructured/pull/908? If so, I'll remove from here and the README

Reply:
Correct, the `unstructured` library now supports 3.11. You can drop that line from the docs and readme.